kit/brain_board/python_libraries.md

```diff
@@ -3,74 +3,47 @@ layout: page
 title: Available Python Libraries
 ---
 
+<!-- Simulator libraries https://github.com/srobo/competition-simulator/blob/main/libraries.txt -->
+<!-- Robot image libraries https://github.com/srobo/robot-image/blob/main/libraries.txt -->
+<!-- Robot image requirements https://github.com/srobo/robot-image/blob/main/requirements.txt -->
```

Comment:
These urls don't work; https://github.com/srobo/robot-image/blob/main/files/python/libraries.txt is what I've been referring to for the simulator.

Reply:
Fixed c850300